src/modules/users/components/TokenActivation/TokenActivationContent/ChangeTokenStateForm.tsx

```diff
@@ -0,0 +1,162 @@
+import { BigNumber, bigNumberify } from 'ethers/utils';
+import { FormikProps } from 'formik';
+import moveDecimal from 'move-decimal-point';
+import * as yup from 'yup';
+
+import Button from '~core/Button';
+import { ActionForm, Input } from '~core/Fields';
+import Numeral from '~core/Numeral';
+
+import { UserToken } from '~data/generated';
+import { ActionTypes } from '~redux/index';
+import { pipe, mapPayload } from '~utils/actions';
+
+import styles from './TokenActivationContent.css';
+
+const MSG = defineMessages({
+  changeState: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.changeState',
+    defaultMessage: 'Change token state',
+  },
+  activate: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.activate',
+    defaultMessage: 'Activate',
+  },
+  withdraw: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.withdraw',
+    defaultMessage: 'Withdraw',
+  },
+  balance: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.balance',
+    defaultMessage: 'balance: {tokenBalance}',
+  },
+});
+
+const validationSchema = yup.object({
+  amount: yup.number().required().moreThan(0),
+});
+
+type FormValues = {
+  amount: number;
+};
+
+export interface ChangeTokenStateFormProps {
+  token: UserToken;
+  tokenDecimals: number;
+  activeTokens: BigNumber;
+  inactiveTokens: BigNumber;
+}
+
+const ChangeTokenStateForm = ({
+  token,
+  tokenDecimals,
+  activeTokens,
+  inactiveTokens,
+}: ChangeTokenStateFormProps) => {
+  const [isActivate, setIsActivate] = useState(true);
+
+  const formAction = useCallback(
+    (actionType: '' | '_ERROR' | '_SUCCESS') =>
+      isActivate
+        ? ActionTypes[`USER_DEPOSIT_TOKEN${actionType}`]
+        : ActionTypes[`USER_WITHDRAW_TOKEN${actionType}`],
+    [isActivate],
+  );
+
+  const transform = useCallback(
+    pipe(
+      mapPayload(({ amount }) => {
+        // Convert amount string with decimals to BigInt (eth to wei)
+        const formtattedAmount = bigNumberify(
+          moveDecimal(amount, tokenDecimals),
+        );
+
+        return { amount: formtattedAmount };
+      }),
+    ),
+    [],
+  );
+
+  return (
+    <div className={styles.changeTokensState}>
+      <div className={styles.changeStateTitle}>
+        <FormattedMessage {...MSG.changeState} />
+      </div>
+      <div className={styles.changeStateButtonsContainer}>
+        <div className={isActivate ? styles.activate : styles.activateInactive}>
+          <Button
+            appearance={{ theme: isActivate ? 'primary' : 'white' }}
+            onClick={() => setIsActivate(true)}
+            text={MSG.activate}
+          />
+        </div>
+        <div className={isActivate ? styles.withdrawInactive : styles.withdraw}>
+          <Button
+            appearance={{ theme: !isActivate ? 'primary' : 'white' }}
+            onClick={() => setIsActivate(false)}
+            text={MSG.withdraw}
+          />
+        </div>
```

Comment:
Colors for the "inactive" button are way **stronger** than the spec.

They need to be a softer shade of grey _(unless this was changed with product after the fact?)_

Reply:
yes, I was missing opacity!